test/integration/test-submit-http.js

```diff
@@ -0,0 +1,53 @@
+var common = require('../common');
+var assert = common.assert;
+var FormData = require(common.dir.lib + '/form_data');
+var correctHostName = 'npmjs.org';
+var correctPort = 80;
+
+// testing default http port
+
+// check params as string
+testRequest('http://npmjs.org/');
+
+testRequest('http://npmjs.org:80/');
+
+// check params as object
+testRequest({protocol: 'http:', hostname: 'npmjs.org', pathname: '/'});
+
+testRequest({port: 80, hostname: 'npmjs.org', pathname: '/'});
+
+testRequest({port: 80, protocol: 'http:', hostname: 'npmjs.org', pathname: '/'});
+
+// --- Santa's little helpers
+
+function testRequest(params)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['testRequest' is defined but never used.](https://www.codacy.com/app/form-data/form-data/file/876417302/issues/source?bid=2805153&fileBranchId=2805189#l23)